### ISSUE TITLE
schema.org implementation

### ISSUE THREAD

**Issue by epgeroy** [daxslab/yii2-taggedview#2]
Feature request: Add [schema.org](http://schema.org) property auto-mapping implementation for view models.

**Comment by glpzzz** [comment]
can you provide an example? This is more about the page, the document, not a certain block on it. Can you confirm there is some scheme.org tagging that can be applied on html > head?

**Comment by epgeroy** [comment]
really!? I already forgot XD

**Comment by epgeroy** [comment]
I think it was about adding [schema.org](schema.org) data to the view models. If you see [https://schema.org/docs/schemas.html](https://schema.org/docs/schemas.html) you'll find common metadata sachems for Organization, Local Business, Restaurants, etc. Adding this metadata to the HTML will help SEO and some crawlers do a good job consequently increasing visibility of the section. There are plenty of examples in the links.

**Comment by glpzzz** [comment]
@epgeroy yes, this is what i know. I dont know about tagging the view itself
